Add a faculty member to PolyU csrankings-h.csv

elainechowpolyu opened the pull request:
Add an existing faculty to The Hong Kong Polytechnic University.

# Contributing to CSrankings

Thanks for contributing to CSrankings! Please read and indicate you agree with **all** these guidelines to getting your pull request accepted. Note that pull requests may take some time to get merged (please don't contact us for at least three months for updates, unless you are a sponsor - see below).

**If you find CSrankings useful, [please consider becoming a sponsor](https://github.com/sponsors/csrankings).**

**NOTE:** IF YOU DO NOT FOLLOW THE STEPS BELOW, YOUR PULL REQUEST WILL BE SUMMARILY REJECTED.
You must read and check **all** the boxes below by filling them in with an X or your PR will be rejected.

**The Basics**

- [x] All pull requests and issues must come from non-anonymous accounts. Make sure your GitHub profile contains your full name.

- [x] Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv").

- [x] Combine multiple updates to a single institution into a **single PR.**

- [x] Only submit one pull request per institution.

- [x] Do not modify any files except `csrankings-[a-z].csv` or (if needed) `country-info.csv` and `old/industry.csv` (see below).

- [x] Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use the GitHub user interface or a text editor like emacs or NotePad instead.

- [x] Insert new faculty **in alphabetical order** (not at the end) in the appropriate `csrankings-[a-z].csv` files. **Do not modify `csrankings.csv`, which is auto-generated.**

- [x] Check to make sure that you have no spaces after commas, or any missing fields.

- [x] Check to make sure the home page is correct.

- [x] Make sure the Google Scholar IDs are just the alphanumeric identifier (not a URL or with `&hl=en`).

- [x] Check to make sure the name corresponds to the DBLP entry (look it up at http://dblp.org).

- [x] If a faculty member is not in a CS department or similar, include a comment explaining how they meet the inclusion criteria (see below).

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. **Provide justification, pointing to specific home pages
showing how faculty not in a CS department meet the inclusion criteria,
e.g. showing a courtesy appointment in CS.** Faculty must also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put `NOSCHOLARPAGE`.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire CS faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**); include disambiguation suffixes like `0001` as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, *all of them need to be listed*. Do not update `dblp-aliases.csv`.

- [x] If the institution you are adding is not in the US,
update `country-info.csv` and add *all* of the faculty in the CS department.

**(Advanced) Quick contribution via a shallow clone** 

We recommend that you use the GitHub web user interface to make changes. However, it may be more convenient
to clone the repository for larger-scale changes.

However, a full clone of the CSrankings repository is almost 2GB.  To
contribute a change without creating a full local clone of the
CSrankings repo, you can perform a _shallow clone_.  To do so, follow these
steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)

1. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.

1. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to make another contribution and some time has passed, perform these steps again, creating a fresh fork and shallow clone.

elainechowpolyu committed:
Add a faculty member to PolyU csrankings-h.csv

Add an existing faculty to The Hong Kong Polytechnic University.

elainechowpolyu committed:
Update affiliation of Man Ho Au (aka Man Ho Allen Au) of PolyU csrankings-m.csv

elainechowpolyu committed:
Add a faculty to PolyU csrankings-q.csv

Add an existing faculty to PolyU.

elainechowpolyu commented:
Hi, there are three commits in total and it seems that checks for the first two commits have not been done. So it cannot proceed to commit validation. Would you please help? Or please advise me of any steps I need to take to proceed. Many thanks.

emeryberger commented:
> All pull requests and issues must come from non-anonymous accounts. Make sure your GitHub profile contains your full name.

Validation succeeded (but was incomplete for some reason), but you still need to add your full name to your Github profile and address the issues below.

emeryberger commented on csrankings-h.csv:
```diff
@@ -58,6 +58,7 @@ Haibin Kan,Fudan University,http://www.cs.fudan.edu.cn/?page_id=14441,NOSCHOLARP
 Haibin Ling,Stony Brook University,https://www3.cs.stonybrook.edu/~hling,jWx3qtIAAAAJ
 Haibing Guan,Shanghai Jiao Tong University,http://www.cs.sjtu.edu.cn/~hbguan,NOSCHOLARPAGE
 Haibo Chen 0001,Shanghai Jiao Tong University,http://ipads.se.sjtu.edu.cn/~hbchen,qd9xSkYAAAAJ
+Haibo Hu 0001,The Hong Kong Polytechnic University,https://haibohu.org/,N3DjoZYAAAAJ&hl
```
Scholar ID needs to have &hl removed; faculty member not apparently in CS and not obviously eligible for inclusion.

https://www.polyu.edu.hk/comp/people/academic-staff/

elainechowpolyu replied:
I have removed the '&hl'. Although Prof. Hu and Dr. Ye are in the Department of Electrical and Electronic Engineering of PolyU, they focus on database and security research. They frequently publish papers in top computer science conferences listed in CSRankings, such as VLDB, ICDE, S&P, SIGMOD, and NeurIPS. 

https://www.polyu.edu.hk/eee/people/academic-staff-and-teaching-staff/

emeryberger replied:
That does not speak to the inclusion criteria. They need to be able to solely supervise PhD students for a PhD in Computer Science.

elainechowpolyu replied:
Thank you for your comment and we noted the inclusion criteria. Could you kindly assist by updating only the affiliation change for Prof. Man Ho Au (also known as Man Ho Allen Au) from The University of Hong Kong (HKU) to The Hong Kong Polytechnic University (PolyU), and omit the other two amendments concerning Prof. Hu and Dr. Ye?

emeryberger commented on csrankings-q.csv:
```diff
@@ -111,6 +111,7 @@ Qingjiang Shi,Tongji University,https://sse.tongji.edu.cn/Data/View/3555,8xoKeR0
 Qingkai Zeng 0002,Nanjing University,https://cs.nju.edu.cn/58/1a/c2639a153626/page.htm,NOSCHOLARPAGE
 Qingling Duan,Queen’s University,https://dbms.queensu.ca/faculty/duan_qingling,NOSCHOLARPAGE
 Qingming Huang,Chinese Academy of Sciences,http://people.ucas.edu.cn/~0003060?language=en,J1vMnRgAAAAJ
+Qingqing Ye 0001,The Hong Kong Polytechnic University,https://www.qingqingye.net/research/,u9xzOh4AAAAJ
```
Not in CS: https://www.polyu.edu.hk/comp/people/academic-staff/.

elainechowpolyu replied:
Although Prof. Hu and Dr. Ye are in the Department of Electrical and Electronic Engineering of PolyU, they focus on database and security research. They frequently publish papers in top computer science conferences listed in CSRankings, such as VLDB, ICDE, S&P, SIGMOD, and NeurIPS. 

https://www.polyu.edu.hk/eee/people/academic-staff-and-teaching-staff/

emeryberger replied:
See comment above.

elainechowpolyu replied:
See comment above. Many thanks.

elainechowpolyu committed:
Update csrankings-h.csv

elainechowpolyu committed:
Update csrankings-h.csv - v2

elainechowpolyu committed:
Merge branch 'emeryberger:gh-pages' into elainechowpolyu-patch-2

emeryberger committed:
Update csrankings-q.csv

Removed one ineligible faculty member

emeryberger committed:
Update csrankings-m.csv

Removed ineligible faculty member

emeryberger committed:
Update csrankings-m.csv

Updated institution

emeryberger committed:
Update csrankings-h.csv

Removed Haibo Hu 0001,The Hong Kong Polytechnic University,https://haibohu.org/,N3DjoZYAAAAJ
